src/clj/game/cards/resources.clj

```diff
@@ -1666,9 +1666,10 @@
 (defcard "Net Mercur"
   {:abilities [{:msg "gain 1 [Credits]"
                 :async true
+                :req (req (pos? (get-counters (get-card state card) :credit)))
                 :effect (req (add-counter state side card :credit -1)
                              (wait-for (gain-credits state side eid 1)
-                                       (trigger-event-sync state side eid :spent-credits-from-card card)))}]
+                                       (trigger-event-sync state side (make-eid state eid) :spent-credits-from-card card)))}]
```

Comment:
ah fuck, I fucked this up. I'm sorry. It should be the other way around. Change `gain-credits` to call `make-eid`, and pass the `eid` directly into `trigger-event-sync`